packages/frontend/src/components/ListingDetail.tsx

```diff
@@ -221,7 +225,15 @@ export default function ListingDetail() {
           <section className="flex gap-4 flex-col bg-white mt-5 md:mt-0 rounded-md md:w-2/5 p-4">
             <div>
               <h3 className=" ">Description</h3>
-              <p data-testid="description">{listing.Metadata.Description}</p>
+              <section data-testid="description">
+                {splitTextByNewlines(listing.Metadata.Description).map(
+                  (line: string, index: number) => (
+                    <p key={`description-${index}`} className="min-h-[1ch]">
```

Comment:
![pr525-2025-05-19](https://github.com/user-attachments/assets/fdc55e79-ac9d-4ca8-a920-44ebc1e14c92)
just a minor quibble / react learning Q: is the `key` property special somehow?